From a4e8bcb3e4f01c41cc8257112e77b81615787b36 Mon Sep 17 00:00:00 2001
From: Matt Fleming <matt.fleming@intel.com>
Date: Thu, 20 Feb 2014 12:59:43 +0000
Subject: [PATCH 2/2] BaseTools: Don't override LFLAGS

The linker flags may need to be customized and the usual way to do this
is from the command line. Unfortunately, it's not possible currently
because the LFLAGS assignment in header.makefile trashes any inherited
value.

Use a conditional assignment instead.

Signed-off-by: Matt Fleming <matt.fleming@intel.com>

diff --git a/BaseTools/Source/C/Makefiles/header.makefile b/BaseTools/Source/C/Makefiles/header.makefile
index df79d40..4b13b43 100644
--- a/BaseTools/Source/C/Makefiles/header.makefile
+++ b/BaseTools/Source/C/Makefiles/header.makefile
@@ -42,7 +42,7 @@ endif
 INCLUDE = $(TOOL_INCLUDE) -I $(MAKEROOT) -I $(MAKEROOT)/Include/Common -I $(MAKEROOT)/Include/ -I $(MAKEROOT)/Include/IndustryStandard -I $(MAKEROOT)/Common/ -I .. -I . $(ARCH_INCLUDE) 
 CPPFLAGS = $(INCLUDE)
 CFLAGS += -MD -fshort-wchar -fno-strict-aliasing -fno-merge-constants -nostdlib -Wall -Werror -c -g
-LFLAGS =
+LFLAGS ?=
 
 ifeq ($(ARCH), IA32)
 #
-- 
1.8.3.2

